src/_shared/scss/_adverts-soulmates.scss

```diff
@@ -188,7 +188,11 @@
 .advert--soulmate {
     &:not(.advert--manual) {
         .advert__image-container {
-            height: 7 * $gs-baseline;
+            height: 10 * $gs-baseline;
```

Comment:
I set the height so that usernames will align horizontally no matter what, knowing that most (if not all) profile pictures are taller than 120px. Then the image will fill its container but not more.